### PR TITLE
I can post now

### PR DESCRIPTION
**Pull request by shwetzpatil**
User can create new post.

### CODE REVIEW

**Comment by shwetzpatil** [comment]
[Posts can be created](https://trello.com/c/V1nWM8CM/13-posts-can-be-created)

File: src/main/js/posts/postsBuilder.js [hunk 1]
```diff
@@ -9,8 +9,12 @@ class PostsBuilder extends React.Component {
   }
 
   componentDidMount() {
-    client({method: 'GET', path: '/api/posts'}).then(response => {
-      this.setState({posts: response.entity._embedded.posts});
+    // client({method: 'GET', path: '/api/posts'}).then(response => {
+    //   this.setState({posts: response.entity._embedded.posts});
+    // });
```
Comment (TinyGobby) [inline src/main/js/posts/postsBuilder.js:14]: Don't forget to remove commented code before committing

File: src/main/java/com/makersacademy/acebook/controller/PostController.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    @GetMapping("/posts")
+    public List<Post> listPosts(){
+        // (List) will help to convert the return value of
+        // findAll to List (similar to Array) type.
+        return (List) postRepository.findAll();
+    }
+
+    @PostMapping("/posts")
+    public void createPost(@RequestBody Post post){
+        Post savedPost = postRepository.save(post);
```
Comment (TinyGobby) [inline src/main/java/com/makersacademy/acebook/controller/PostController.java:27]: Excellent use of the crudRepository methods inherited by postRepository

File: src/main/js/posts/inputPost.js [hunk 3]
```diff
@@ -13,8 +35,15 @@ class InputPost extends React.Component {
...
-                    <button type="submit" name="submitPost">Take a Longshot</button>
+                    <textarea name="message"
+                              placeholder="Don't worry, be happy"
+                              value={this.state.content}
+                              onChange={this.handleChange} />
+                    <button type="submit"
+                            name="submitPost"
+                            onClick={this.createPost}>
+                        Take a Longshot
+                    </button>
```
Comment (TinyGobby) [inline src/main/js/posts/inputPost.js:46]: Nicely formatted, very easily readable

**Review by TinyGobby — APPROVED** [review]
Really nice work Shweta

**Comment by TinyGobby** [comment]
[Post can be displayed](https://trello.com/c/TZP1P4TN/36-post-can-be-displayed)

File: src/main/java/com/makersacademy/acebook/controller/PostController.java (hunk 2 above)
Comment (racldn) [reply src/main/java/com/makersacademy/acebook/controller/PostController.java:27]: Shweta - can catch up on this separately after the retro?

**Review by racldn — APPROVED** [review]

**Pull request merged**